### PR TITLE
Interview web sockets

### PR DESCRIPTION
**Pull request by AndresNeisa**
The interview UI is using sockets to reflect the changes between users.

### CODE REVIEW

File: client/src/pages/Interview.js [hunk 1]
```diff
@@ -156,6 +186,17 @@ const Interview = (props) => {
...
     setResults(result.data.stderr || result.data.stdout);
+    socket.emit("compile", result.data.stderr || result.data.stdout);
+  };
+
+  const handleCodeChange = (code) => {
+    setCode(code);
+    socket.emit("code", code);
+  };
+
+  const handleToggleChange = (event, newLanguaje) => {
```
Comment (bonnieli) [inline client/src/pages/Interview.js]:
```suggestion
  const handleToggleChange = (event, newLanguage) => {
```

Comment (AndresNeisa) [reply client/src/pages/Interview.js]: Change has already been made

File: client/src/pages/Interview.js [hunk 2]
```diff
@@ -140,6 +154,20 @@ const Interview = (props) => {
...
+        setCode(code);
+      });
+      socket.on("compile", (result) => {
+        setResults(result);
+      });
+      socket.on("language", (language) => {
+        setLanguage(language);
+      });
+    }
+  }, []);
```
Comment (bonnieli) [inline client/src/pages/Interview.js:175]: you'll want to turn these listeners off ^ `code` `compile` `language` or else if you go back to the main page and enter this Interview component again, it'll keep replicating these listeners

File: client/src/pages/Interview.js [hunk 3]
```diff
@@ -173,6 +202,20 @@ const Interview = (props) => {
     });
 
     setResults(result.data.stderr || result.data.stdout);
+    socket.emit("compile", result.data.stderr || result.data.stdout);
+  };
+
+  const handleCodeChange = (code) => {
+    setCode(code);
+    socket.emit("code", code);
+    console.log(props.navHeight);
```
Comment (bonnieli) [inline client/src/pages/Interview.js]: remove when done

Comment (AndresNeisa) [reply client/src/pages/Interview.js]: Console log has been removed

**Review by bonnieli — APPROVED** [review]

**Pull request merged**